pkg/universe.dagger.io/x/hardy4yooz@gmail.com/notify/discord/discord.cue

```diff
@@ -0,0 +1,40 @@
+package discord
+
+import (
+	"dagger.io/dagger"
+	"encoding/json"
+	"universe.dagger.io/alpine"
+	"universe.dagger.io/docker"
+)
+
+// Send message to discord channel
+#Send: {
+	// Discord Webhook URL
+	webhookUrl: dagger.#Secret
+
+	// Discord json parameters
+	parameters: #Paramter
+
+	_image: alpine.#Build & {
+		packages: {
+			curl: {}
+		}
+	}
+
+	run: docker.#Run & {
```

Comment:
Nit: using `bash.#Run` is more accurate here because you run a shell script

Reply:
Yes it is, embedding is really powerful and that basically helps reducing level of depth and details in high level implementation.